packages/vm/src/evm/evm.ts

```diff
@@ -298,6 +298,20 @@ export default class EVM {
     // Reduce tx value from sender
     await this._reduceSenderBalance(account, message)
 
+    if (this._vm._common.isActivatedEIP(3860)) {
+      if (message.data.length > this._vm._common.param('vm', 'maxInitCodeSize')) {
+        return {
+          gasUsed: message.gasLimit,
+          createdAddress: message.to,
+          execResult: {
+            returnValue: Buffer.alloc(0),
+            exceptionError: new VmError(ERROR.INITCODE_SIZE_VIOLATION),
+            gasUsed: message.gasLimit,
```

Comment:
When I read the EIP, it says that if a "create transaction exceeds `MAX_INITCODE_SIZE`, transaction is invalid."  Does that mean that we do an exceptional abort and consume all gas or that the transaction is just rejected and not included in the block?

Reply:
I went ahead and added a new EIP-3860 test file to the `api/EIPs` section and added a real basic test to trigger this error so we can bump test coverage.

Reply:
It is the latter, I remember having the same question and it is answered somewhere, but I cannot find it again 😅 So, the block is invalid if you include such transaction and nodes will reject that broadcasted block.